README.md

```diff
@@ -1,27 +1,53 @@
 # Role Name
 
-A brief description of the role goes here.
-
+This role sets up fail2ban.
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here.
-For instance, if the role uses the EC2 module or depends on other Ansible roles, it may be a good idea to mention in this section that the boto package is required.
-
+No requirements, but can be used in combination with the [nginx](https://github.com/stuvusIT/nginx) role. 
 
 ## Role Variables
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role.
-Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
-
-```yml
-```
+### General
+
+| Name | Required/Default | Description |
+|------|------------------|-------------|
+| `fail2ban_bantime` | `3600` | Time to ban hosts. Can be overwritten for individual jails. |
+| `fail2ban_findtime` | `600` | If a hosts exceeds `fail2ban_maxretry` violations within this timeframe it will get banned. Can be overwritten for individual jails. |
+| `fail2ban_maxretry` | `3` | Maximum violations until ban. Can be overwritten for individual jails. |
+| `fail2ban_enabled` | `False` | On default all jails are disabled and must be enabled individually. |
```

Comment:
```suggestion
| `fail2ban_enabled` | `False` | Whether to enable all jails by default. |
```

Wouldn't `True` be a better default? In most cases, one wants to use the jails he configured.

Reply:
Wouldn't True be a better default? In most cases, one wants to use the jails he configured.

Reply:
Well that's the default in the generic fail2ban config. There are a lot of predefined jails and you can enable them one by one if you need them. Or alternativley if you want to enable all of them you set the `fail2ban_enabled` variable to `true`.

I think it's better to not enable every jail on default, because jails might require specific configurations depending on the host setup (e.g. reverse proxy) and can potentially have a severe impact on the hosts operation if misconfigured.